web/src/components/apps/AppVersionHistory.tsx

```diff
@@ -216,6 +221,7 @@ class AppVersionHistory extends Component<Props, State> {
       viewLogsErrMsg: "",
       yamlErrorDetails: [],
       shouldShowUpgradeServiceModal: false,
+      isStartingUpgradeService: {},
```

Comment:
how about naming it this since it's an object, not a boolean?
```suggestion
      upgradeService: {},
```